### DISCUSSION TITLE
Is Containerfile supported by the docker manager?

### DISCUSSION THREAD

**Discussion by lmilbaum** [renovatebot/renovate#20760] [Request Help] [answered]
### How are you running Renovate?

Mend Renovate hosted app on github.com

### If you're self-hosting Renovate, select which platform you are using.

None

### Was this something which used to work for you, and then stopped?

I never saw this working

### Wanted end result.

Renovate to pick up dependencies from Containerfile as with Dockerfiles

### What you tried so far.

Adding the following to config:

```
  "dockerfile": {
    "fileMatch": [
      "(^|/|\\.)Containerfile$",
      "(^|/)Containerfile[^/]*$"
    ]
  }
```

### Relevant debug logs

<details><summary>Logs</summary>

```
Copy/paste the relevant log(s) here, between the starting and ending backticks
```

</details>

**Comment by rarkins** [comment, marked as answer]
Only if they follow the same format as `Dockerfile`

**Reply by lmilbaum** [reply]
They do but renovate is not picking it up unless the config file is updated. I would suggest that the default would be to pick up both options: files named Dockerfile and Containerfile.

**Reply by rarkins** [reply]
I think it's ok to extend the default `fileMatch` for as long as the syntax of the two are the same where it matters to us. And naturally the manager name renames `dockerfile`

**Reply by lmilbaum** [reply]
Makes sense to me.  I will start working on it.
